src/Utils.js

```diff
@@ -12,6 +12,10 @@ import {
   REACT15,
 } from './version';
 
+export const ITERATOR_SYMBOL = (
+  typeof Symbol === 'function' && Symbol.iterator
+) || '@@iterator';
```

Comment:
To avoid the complexity of having to conditionally define a method on a class when there's not `Symbol.iterator` available, this falls back to `'@@iterator'`. I don't think this should affect environments where neither is supported.